include/zephyr/pm/policy.h

```diff
@@ -246,6 +254,16 @@ void pm_policy_device_power_lock_get(const struct device *dev);
  */
 void pm_policy_device_power_lock_put(const struct device *dev);
 
+#ifdef CONFIG_PM_LOW_LATENCY_EVENTS
+/**
+ * @brief Returns the ticks until the next low latency events
+ *
+ * If a low latency event is registred, it will return the number of ticks
+ * until the next low latency event, otherwise it returns -1
+ */
+int32_t pm_policy_next_low_latency_event_ticks(void);
+#endif /* CONFIG_PM_LOW_LATENCY_EVENTS */
```

Comment:
is this a function meant to be public?

Reply:
This function is called from pm.c to check the next low latency event